app/src/main/java/com/example/wrk/fragments/ProfileFragment.java

```diff
@@ -236,7 +233,7 @@ private double calcPercentage() throws ParseException {
             double difference = currentUser.getInt("workoutsThisMonth") - averageWorkoutsThisMonth;
             percentDifference = (difference / averageWorkoutsThisMonth) * 100;
         }
-        return percentDifference;
+        return Math.round(percentDifference * 10.0) / 10.0; // round to one decimal
```

Comment:
would be nice to point out in the PR that you are also modifying small thing unrelated to the main PR content

Reply:
Noted.